_pytest/terminal.py

```diff
@@ -42,6 +42,10 @@ def pytest_addoption(parser):
                      action="store", dest="tbstyle", default='auto',
                      choices=['auto', 'long', 'short', 'no', 'line', 'native'],
                      help="traceback print mode (auto/long/short/line/native/no).")
+    group._addoption('--show-capture',
+                     action="store", dest="showcapture",
+                     choices=['no', 'stdout', 'stderr'],
```

Comment:
You are missing the `"both"` option (which should also be the default).

Reply:
If I add "both" option it is not change a code, because "both" means without this option. is it really needed?

Reply:
We do, the default value for options must be defined, all other options do,  for example --capture, --color, any other multiple choice option really.